src/components/scenes/SendScene.js

```diff
@@ -314,18 +315,19 @@ class SendComponent extends React.PureComponent<Props, State> {
         const displayAmount = bns.div(nativeAmount, cryptoDisplayDenomination.multiplier, UTILS.DIVIDE_PRECISION)
         const exchangeAmount = bns.div(nativeAmount, cryptoExchangeDenomination.multiplier, UTILS.DIVIDE_PRECISION)
         const fiatAmount = convertCurrencyFromExchangeRates(exchangeRates, selectedCurrencyCode, guiWallet.isoFiatCurrencyCode, parseFloat(exchangeAmount))
-        amountSyntax = `${displayAmount || '0'} ${cryptoDisplayDenomination.name} = (${fiatSymbol} ${fiatAmount.toFixed(2) || '0'})`
+        cryptoAmountSyntax = `${displayAmount || '0'} ${cryptoDisplayDenomination.name}`
```

Comment:
We should probably get into the habit of using `??` instead of `||` for fallback values like this. It's more explicit about what the intent it, and React Native + Flow support it.